sql/mob_family_system.sql

```diff
@@ -537,6 +537,7 @@ INSERT INTO `mob_family_system` VALUES (493,'Ashrakk',9,'Demon',1,40,90,140,6,3,
 INSERT INTO `mob_family_system` VALUES (494,'Sekhmet',6,'Beast',1,60,94,90,1,1,4,2,6,6,2,3,3,3,3,1,1,1,1,1,1,1,1.25,0.875,1,1,1,5);
 INSERT INTO `mob_family_system` VALUES (495,'Astral_Flow_Pet',5,'Avatar',0,40,100,120,3,3,3,3,3,3,3,3,3,3,3,1,1,1,1,1,1,1,1,1,1,1,1,0);
 INSERT INTO `mob_family_system` VALUES (496,'Mandragora',17,'Plantoid',0,40,107,120,3,3,4,5,4,4,2,3,3,3,3,1,1.25,1,1,1.25,1.25,1.25,1,1.25,1,0.875,1.25,4); -- for chaneque unique TP move
+INSERT INTO `mob_family_system` VALUES (497,'Poroggo',7,'Beastmen',0,40,70,140,6,3,6,3,1,5,4,3,3,3,3,1,1,1,1,1,1.125,1,1,1.125,0.5,0.5,1,6);

```

Comment:
Oh, just read on wiki he does no tp moves at all and casts no spells, if that's the only reason for the new family we can actually just set this in his script:
`mob:SetMobAbilityEnabled(false);` And with no spell list he won't try to cast.

Otherwise, ignore this post.

There was an NM I put in Upper Deklfutts Tower I think, when I was new to the project..I need to go back and do the same thing to him. He error's the console log about trying to use TP when he has no skills for his FamilyID whenever someone fights him :frowning: